docs/release/2.x.rst

```diff
@@ -331,6 +331,9 @@ method name consistent with GEOS or PostGIS:
 * :func:`.shortest_line` (``ops.nearest_points``)
 * :func:`.is_valid_reason` (``validation.explain_validity``)
 
+Functions with additional parameters introduced with new GEOS versions:
+* :func:`.line_merge`  (GEOS > 3.11) added `directed` parameter.
```

Comment:
I think it is fine to add it to the first list of new functions from GEOS (it's indeed not a new function in shapely, but is _is_ exposing a new GEOS function ..). If it is worded as "added directed parameter" then that should be clear I think